static/js/components/VideoPlayer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React from 'react';
+
+/**
+ * Before using, import the following (as in ../entry/videojs):
+ *     require('videojs5-hlsjs-source-handler');
+ *     import 'expose-loader?videojs!video.js';
+ * These are not imported here due to babel issues
+ * with expose-loader & a hlsjs dependency that cause
+ * tests to fail.
+ */
+
+export default class VideoPlayer extends React.Component {
+  /*
+    Basic component based on https://github.com/videojs/video.js/blob/master/docs/guides/react.md
+    TODO: Add more configuration options, including optional rendering of <TextTrack> elements for subtitles.
+   */
+  componentDidMount() {
+    // instantiate video.js
+    this.player = videojs(this.videoNode, this.props, function onPlayerReady() { // eslint-disable-line no-undef
```

Comment:
Since `videojs` is a global here can you add `/* SETTINGS videojs: false */` to the top line of this file? At some point we should configure the linter to check for this